src/job-posting/dto/create-job.dto.ts

```
@@ -0,0 +1,123 @@
+  @IsOptional()
+  status?: JobStatus = JobStatus.ACTIVE
+
+  @IsEnum(ExperienceLevel)
+  @IsOptional()
+  experienceLevel?: ExperienceLevel = ExperienceLevel.MID
+
+  @IsNumber()
+  @IsOptional()
+  @Min(0)
+  @Type(() => Number)
+  salaryMin?: number
```

ESLint

[error] Insert `;`

(prettier/prettier)